ssget

```diff
@@ -37,6 +37,7 @@ set -e
 THIS_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" &>/dev/null && pwd )
 SS_URL="https://sparse.tamu.edu"
 ARCHIVE_LOCATION="${HOME}/.config/ssget"
+CUSTOM_ARCHIVE=""
 
 mkdir -p "${ARCHIVE_LOCATION}"
```

Comment:
If the archive location is not in ${HOME}, then I think it does not make sense to create a `ssget` folder there.